app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -168,14 +168,10 @@ public static void playOnPopupPlayer(final Context context,
     public static void playOnBackgroundPlayer(final Context context,
                                               final PlayQueue queue,
                                               final boolean resumePlayback) {
-        Toast.makeText(
-                context,
-                PlayerHolder.getInstance().getType() == PlayerType.AUDIO
-                    ? R.string.background_player_already_playing_toast
-                    : R.string.background_player_playing_toast,
-                Toast.LENGTH_SHORT)
-                .show();
-
+        if (PlayerHolder.getInstance().getType() != MainPlayer.PlayerType.AUDIO) {
+            Toast.makeText(context, R.string.background_player_playing_toast, Toast.LENGTH_SHORT)
+                    .show();
+        }
```

Comment:
I'd just remove the `if` and call it a day. This way when starting to play in the background (which could or could not replace an already playing queue, but we don't care) a toast is always shown.
```suggestion
        Toast.makeText(context, R.string.background_player_playing_toast, Toast.LENGTH_SHORT)
                .show();
```

Reply:
Sounds good.

but than it would differ from the popup toast. 
@Stypox Should we fix this also here?

Reply:
I'd say yes

Reply:
Done